cobbler.spec

```diff
@@ -328,20 +326,16 @@ fi
 %post web
 # Change the SECRET_KEY option in the Django settings.py file
 # required for security reasons, should be unique on all systems
-# Choose from multiple characters, but without an ampersand (&).
-RAND_SECRET=$(head /dev/urandom | tr -dc 'A-Za-z0-9!"#$%'\''()*+,-./:;<=>?@[\]^_`{|}~' | head -c 57 ; echo '')
+# Choose from letters and numbers only, so no special chars like ampersand (&).
+RAND_SECRET=$(head /dev/urandom | tr -dc 'A-Za-z0-9!' | head -c 50 ; echo '')
 sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler/web/settings.py
 
 
 %files
 %license COPYING
 %doc AUTHORS.in README.md
 %doc docs/developer-guide.rst docs/quickstart-guide.rst docs/installation-guide.rst
-%if %{_vendor} == "debbuild"
-%config(noreplace) %{_sysconfdir}/cobbler/*
-%else
 %config(noreplace) %{_sysconfdir}/cobbler
-%endif
```

Comment:
config files cannot be directories... If you're going to do this, then enumerate all the config files and mark them as `%config(noreplace)`

Reply:
Reverted to Enno's change.
Will result in some more `dpkg-deb: warning: conffile`.

Reply:
```console
dpkg-deb: warning: conffile '/etc/cobbler/boot_loader_conf' is not a plain file
dpkg-deb: warning: conffile '/etc/cobbler/iso' is not a plain file
dpkg-deb: warning: conffile '/etc/cobbler/power' is not a plain file
dpkg-deb: warning: conffile '/etc/cobbler/reporting' is not a plain file
dpkg-deb: warning: conffile '/etc/cobbler/settings.d' is not a plain file
dpkg-deb: warning: conffile '/etc/cobbler/zone_templates' is not a plain file
dpkg-deb: warning: ignoring 6 warnings about the control file(s)
```